### PR TITLE
feat(validation): Add support for validation of date custom field

### PR DESCRIPTION
**Pull request by ioanpocol**

### CODE REVIEW

File: apps/validate/validate.py [hunk 1]
```diff
@@ -65,6 +67,12 @@ def _validate_type_embed(self, field, value):
         """Allow type media in schema."""
         pass
 
+    def _validate_type_date(self, field, value):
+        try:
+            datetime.strptime(value or '', '%d/%m/%Y')
```
Comment (petrjasek) [inline apps/validate/validate.py]: we use `%Y-%m-%d` usually, why this format here?

Comment (ioanpocol) [reply apps/validate/validate.py]: This was the default format on frontend, I changed on frontend to send always in  %Y-%m-%d format.

**Comment by petrjasek** [comment]
can you add some test where it would check any values?

**Comment by ioanpocol** [comment]
Done

**Pull request merged**